xml/yast2_sw.xml

```diff
@@ -275,6 +275,17 @@
       </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>Services</term>
```

Comment:
Whoever came up with the term "services" for modules and extensions should be punished. :-/

Reply:
Yes, adding yet another term does not really help here. ;)